features/data_converter/binary_protobuf/feature.ts

```diff
@@ -0,0 +1,59 @@
+import { Feature } from '@temporalio/harness';
+import * as proto from '@temporalio/proto';
+import * as assert from 'assert';
+
+// Inject Buffer and Uint8Array into isolate to workaround SDK bug
+// TODO(antlai-temporal) Remove when SDK bug is fixed
+const g = globalThis as any;
+g.Uint8Array = g.constructor.constructor('return globalThis.Uint8Array')();
```

Comment:
I still need to inject Uint8Array (not Buffer) because protobufjs does an instanceof check for Uint8Array when decoding (see reader.js:45  in protobufjs)